components/textbox/textbox.jsx

```diff
@@ -52,7 +58,12 @@ export default class Textbox extends React.Component {
         this.state = {};
 
         this.suggestionProviders = [
-            new AtMentionProvider(this.props.channelId),
+            new AtMentionProvider({
```

Comment:
Ideally, `AtMentionProvider` would be connected itself, but while this is technically possible, it still leaves us without a way to back-propagate pretext changes on props-only changes unless `Textbox` is in on the loop.